x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -24,6 +25,7 @@ public EqlFunctionRegistry() {
         // String
             new FunctionDefinition[] {
                 def(Substring.class, Substring::new, "substring"),
+                def(CIDRMatch.class, CIDRMatch::new, "cidrmatch"),
```

Comment:
I prefer listing the functions in alphabetical order.

Reply:
Yep. Updated.